reconcile/slack_usergroups.py

```diff
@@ -243,6 +243,11 @@ def get_desired_state(slack_map, pagerduty_map):
         workspace_name = workspace['name']
         usergroup = p['handle']
         description = p['description']
+        channel_names = [] if p['channels'] is None else p['channels']
+        if channel_names == []:
```

Comment:
You can just make this `if not channel_names` which is a pretty standard way to handle any falsy values in Python - `None`, `[]`, `0`, etc - unless you really care whether it's one of those specific types (sometimes we care about `None` to indicate something is unset).

Also, I don't even think that you need to make `channel_names` and empty list anymore because you're not passing it in anywhere that expects a list. As mentioned above, checking `if not <object>` will handle `None` just fine.